composer.json

```diff
@@ -33,11 +23,17 @@
     },
     "autoload": {
         "psr-4": {
-            "SilverStripe\\SpamProtection\\": "code/",
+            "SilverStripe\\SpamProtection\\": "src/",
```

Comment:
Does this have any chance for regressions?
Will git automatically recognise the change of directory when we do merge-ups?

Reply:
@wilr Can you please clarify this for me?

Reply:
There isn't any open PR's so it won't cause any issues there.

Reply:
The concern is for if it will result in regressions, or if it will cause issues with merge-ups (e.g. merging up a bugfix from the `4.1` branch to `4`) - nothing to do with PRs.

Reply:
@GuySartorelli It hasn't caused issues on any other repos we've done this migration on so we should be green.